docs/source/openvino/export.mdx

```diff
@@ -109,6 +96,37 @@ Models larger than 1 billion parameters are exported to the OpenVINO format with
 
 </Tip>
 
+
+### Decoder models
+
+```bash
+optimum-cli export openvino --model  meta-llama/Meta-Llama-3-8B --task text-generation-with-past ov_llama/
+```
+
+The `-with-past` suffix enable the re-use of past keys and values. This allows to avoid recomputing the same intermediate activations at each generation step. To export the model without, you will need to remove this suffix.
```

Comment:
I know this is not a change but just moved, but LLMs are so popular and I wonder if the default command to show can be just `optimum-cli export openvino --model meta-llama/Meta-Llama-3-8B ov_llama` and then note that this is the default and that if you do not want to use the K-V cache, you can export the model without it. That makes the default command much easier to remember.

Reply:
updated in https://github.com/huggingface/optimum-intel/pull/794/commits/7cd69a0a8830fef988faa23034768fd294801843